### PR TITLE
load local faqs on development

### PR DESCRIPTION
**Pull request by drewvolz**
load the local faqs when developing instead of rendering remote data

### CODE REVIEW

File: views/faqs/index.js [hunk 1]
```diff
@@ -35,6 +35,11 @@ export class FaqView extends React.Component {
       tracker.trackException(err.message)
       console.warn(err.message)
```
Comment (hawkrives) [inline views/faqs/index.js:37]: Would you add a fallback in here, too, please?

File: views/faqs/index.js [hunk 2]
```diff
@@ -35,6 +35,11 @@ export class FaqView extends React.Component {
...
     }
+
+    if (__DEV__) {
+      html = faqs
```
Comment (hawkrives) [inline views/faqs/index.js:41]: This needs to be faqs.text

**Review by hawkrives — CHANGES_REQUESTED** [review]

File: views/faqs/index.js (hunk 2 above)
Comment (drewvolz) [reply views/faqs/index.js:41]: It's using the text imported from the json above.

`import {text as faqs} from '../../docs/faqs.json'`

Comment (hawkrives) [reply views/faqs/index.js:41]: Ah. excdllent point. Carry on.

Comment (hawkrives) [reply views/faqs/index.js:41]: Thank you for not correcting, autocorrect...

File: views/faqs/index.js (hunk 1 above)
Comment (drewvolz) [reply views/faqs/index.js:37]: Done.

**Review by hawkrives — APPROVED** [review]
LGTM.

**Pull request merged**